test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextJsonWriterTests.cs

```diff
@@ -40,7 +40,7 @@ public JObject Save(DependencyContext dependencyContext)
             IReadOnlyList<RuntimeFallbacks> runtimeGraph = null)
         {
             return new DependencyContext(
-                            target ?? string.Empty,
+                            target ?? "DefaultTarget",
```

Comment:
Hrm... I think I'd almost rather a `null`y value (like `string.Empty`) than some arbitrarily-named default we never expect someone to actually use...